templates/solr.in.sh.j2

```diff
@@ -73,41 +94,92 @@ SOLR_PID_DIR="{{ solr_home }}"
 # If solr.xml is not stored in ZooKeeper, this directory needs to contain solr.xml
 SOLR_HOME="{{ solr_data_dir }}"
 
-# Solr provides a default Log4J configuration properties file in server/resources
+# Path to a directory that Solr will use as root for data folders for each core.
+# If not set, defaults to <instance_dir>/data. Overridable per core through 'dataDir' core property
+#SOLR_DATA_HOME=
+
+# Solr provides a default Log4J configuration xml file in server/resources
 # however, you may want to customize the log settings and file appender location
-# so you can point the script to use a different log4j.properties file
-LOG4J_PROPS="/opt/solr/log4j.properties"
+# so you can point the script to use a different log4j2.xml file
+LOG4J_PROPS={{ {{ solr_installation_dir }} }}/{{ solr_log_config_file }}
```

Comment:
seems that should be only with one level of braces :) `{{ solr_installation_dir }}`

Reply:
yes... you're right!

Changed.